### DISCUSSION TITLE
"[ACTION REQUIRED] Your GitHub account, corwinn, will soon require 2FA"

### DISCUSSION THREAD

**Discussion by corwinn** [community/community#67208] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm receiving this e-mail for a 3rd or a 4th time. Could you please stop spamming me with it? Thank you.
I can't find a way to unsubscribe from "it". It is unwanted e-mail. Please stop sending me one and the same e-mail over and over again. Adhere to the law if you please, please. Thank you.

**Comment by corwinn** [comment]
I received another unwanted e-mail - you're continuing your criminal activity "github". Stop spamming me, please.

**Comment by ghost** [comment]
Github have said that all users currently using Github will have to enable 2FA to continue using the platform, well at least the people who contribute code. They have been gradually rolling this out since March. If you have not done this, then they're probably going to be persistent about it, since eventually it will be a mandatory requirement moving forward.

I get it though! I don't like being spammed with emails either.

**Reply by corwinn** [reply]
This "activity" - sending unwanted emails against my will - is outlawed in my country. If "Microsoft" continues with the spam, I'll notify my local authorities, and the EU ones (since my country is part of the EU). I'm sure that my "notification" will have no direct effect on the arrogance, but I do hope that enough "drops" will create a "wave" one day.

As sure as the rivers reach the seas - I do know now, what these people are.
